tests/memarray/README.md

```diff
@@ -0,0 +1,18 @@
+Expected result
+===============
+
+This application should run a number of tests equal to NUMBER_OF_TESTS (Default 12).
+
+At the beginning of the tests memory usage of each thread is printed.
+At the end of the tests, the threads memory usage is printed once more time.
+
+This test is passed if the memory used by the main thread remains static.
+
+Background
+==========
+
+The module `memarray` is the static memory allocator for RIOT-OS
```

Comment:
IMO this needs re-phrasing. What is a "static memory allocator"?

Reply:
What do you think of "fixed-size blocks allocator"?

Taken from https://en.wikipedia.org/wiki/Memory_management#FIXED-SIZE and I find it clearer than "memory pool allocator"

Reply:
I think "fixed-size block allocator" and "memory pool allocator" both are true, but highlight different properties of the allocator. Also, the fact that it just allocates fixed-size blocks may be the more distinct feature of this implementation, so I'm on your side with this.